qualtrics_link/management/commands/qualtrics.py

```diff
@@ -51,6 +54,8 @@ def handle(self, *args, **options):
             self.update_stats()
         elif options['perform_full_update']:
```

Comment:
Could we explore  removing lines  55-56 , so that command is not accidentally invoked? I would leave the perform_full_update function in place, but may explore removing it here. I am open to leaving as is if we think we may go back to this, but sounded like we don't want bulk updates

Reply:
I'll remove it